packages/amplify_datastore_plugin_interface/test/testData/Comment.dart

```diff
@@ -70,7 +70,7 @@ class Comment extends Model {
     return buffer.toString();
   }
 
-  Comment copyWith({@required String id, Post post, @required String content}) {
+  Comment copyWith({required String id, Post? post, required String content}) {
```

Comment:
copyWith probably won't need `required`

Reply:
thanks.  Let's ignore the dart datastore codegen models for now.  There is a bit more complexity going on there that we can address in a future meeting (scheduled for next Monday) and eventually within the dedicated Datastore NS PR.